modules/administration-guide/partials/con_caching-images-for-faster-workspace-start.adoc

```diff
@@ -57,6 +57,10 @@ endif::[]
 |`NODE_SELECTOR` 
 |Node selector to apply to the Pods created by the DaemonSet 
 |`'{}'`
+
+|`IMAGE_PULL_SECRETS` 
+| List of image pull secrets, in the format `pullsecret1;...` to add to pods created by the daemonset. Those secrets need to be in the same namespace as the image puller and need to be created by a cluster admin.
```

Comment:
```suggestion
| List of image pull secrets, in the format `pullsecret1;...` to add to pods created by the DaemonSet. Those secrets need to be in the same namespace as the image puller and need to be created by a cluster admin.
```
Vale borks on the daemonset https://github.com/eclipse/che-docs/pull/1957/checks?check_run_id=2391883394